.travis.yml

```diff
@@ -355,9 +355,13 @@ script:
   - env PATH=$PATH:`pwd`/src/solvers UBSAN_OPTIONS=print_stacktrace=1 make -C regression/cbmc test-cprover-smt2
   - make -C unit "CXX=${COMPILER} ${EXTRA_CXXFLAGS}" -j2
   - make -C unit test
+  - echo "Running expected failure tests"
+  - make TAGS="[shouldfail]" -C unit test
   - env UBSAN_OPTIONS=print_stacktrace=1 make -C jbmc/regression test-parallel "CXX=${COMPILER} ${EXTRA_CXXFLAGS}" -j2 JOBS=2
   - make -C jbmc/unit "CXX=${COMPILER} ${EXTRA_CXXFLAGS}" -j2
   - make -C jbmc/unit test
+  - echo "Running expected failure tests"
+  - make TAGS="[shouldfail]" -C jbmc/unit test xfail
```

Comment:
??? Why not just use the builtin `!shouldfail` mechanism?

Reply:
Not sure what you mean? The trailing `xfail` was a typo, but this is me passing the appropriate tags to the unit runner

Reply:
This should be `[!shouldfail]`, not `[shouldfail]`. I believe might be fixed by a later commit?